packages/tools/devtools/devtools-view/src/test/Menu.test.tsx

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	type DevtoolsFeatureFlags,
+	DevtoolsFeatures,
+} from "@fluidframework/devtools-core/internal";
+// Normal usage pattern for @testing-library/jest-dom
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import React from "react";
+
+import { MessageRelayContext } from "../MessageRelayContext.js";
+import { Menu, type MenuSelection } from "../components/index.js";
+
+import { assertNoAccessibilityViolations, MockMessageRelay } from "./utils/index.js";
+
+describe("Menu Accessibility Check", () => {
+	const supportedFeatures: DevtoolsFeatureFlags = {
+		telemetry: true,
+		opLatencyTelemetry: true,
+	};
+	const containers = ["Container1", "Container2"];
+
+	const MenuWrapper: React.FC = () => {
+		const [menuSelection, setMenuSelection] = React.useState<MenuSelection | undefined>();
+
+		return (
+			<Menu
+				currentSelection={menuSelection}
+				setSelection={setMenuSelection}
+				containers={containers}
+				supportedFeatures={supportedFeatures}
+			/>
+		);
+	};
+	const mockMessageRelay = new MockMessageRelay(() => {
+		return {
+			type: DevtoolsFeatures.MessageType,
+			source: "OpLatencyTest",
+			data: {
+				features: {
+					telemetry: true,
+					opLatencyTelemetry: true,
+				},
+				devtoolsVersion: "1.0.0",
+				unsampledTelemetry: true,
+			},
+		};
+	});
+
+	it("Menu is accessible", async () => {
+		const { container } = render(
+			<MessageRelayContext.Provider value={mockMessageRelay}>
+				<MenuWrapper />
+			</MessageRelayContext.Provider>,
```

Comment:
For test purposes I think the relay can also go inside the wrapper. We'd expect every test in this file to have to do this same thing, right?

Reply:
Yeah good point